includes/wizards/class-updates-wizard.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Newspack's Updates Wizard
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use \WP_Error, \WP_Query;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once NEWSPACK_ABSPATH . '/includes/wizards/class-wizard.php';
+
+/**
+ * Easy interface for setting up general store info.
+ */
+class Updates_Wizard extends Wizard {
+
+	/**
+	 * The slug of this wizard.
+	 *
+	 * @var string
+	 */
+	protected $slug = 'newspack-updates-wizard';
+
+	/**
+	 * The capability required to access this wizard.
+	 *
+	 * @var string
+	 */
+	protected $capability = 'manage_options';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+		add_action( 'rest_api_init', [ $this, 'register_api_endpoints' ] );
+	}
+
+	/**
+	 * Get the name for this wizard.
+	 *
+	 * @return string The wizard name.
+	 */
+	public function get_name() {
+		return \esc_html__( 'Updates', 'newspack' );
+	}
+
+	/**
+	 * Get the description of this wizard.
+	 *
+	 * @return string The wizard description.
+	 */
+	public function get_description() {
+		return \esc_html__( 'Updates to the Newspack plugins and themes.', 'newspack' );
```

Comment:
```suggestion
		return \esc_html__( 'Updates to the Newspack plugins and theme.', 'newspack' );
```

Reply:
Done in bbd8ce88b6765d22ecc9ebd311e4262f78bc9a69